FirstBrainwaveDroneDeploymentDocs/deeplearningpytorchpredictory.in_the_state_of_our_testing.py

```
@@ -0,0 +1,70 @@
+            print("======================\n\n\n\n\n")
+
+            print(json.dumps(predicted_classes.tolist()))
+
+            
+            print("========================\n\n\n\n\n")
+
+        # Return the first label
+        return pred_labels[0]
+    
+    if __name__ == "__main__":
+        DeeplearningPyTorchPredictor()
```

Ruff (0.14.7)

Undefined name `DeeplearningPyTorchPredictor`

(F821)